src/kernelManager/kernelChildNodeProvider.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { Event, TreeItem } from 'vscode';
+import { IActiveLocalKernelTreeNode, IActiveRemoteKernelTreeNode, ICustomNodeFromAnotherProvider } from './types';
+
+export interface IActiveKernelChildNodesProvider {
```

Comment:
Added ability to create more extension points to add child nodes under each kernel.
Plan is to add more extensions that allow users to do stuff with kernels (live and kernel specs).

Reply:
Also helps isolate each functionality instead of dumping everything into the kernel manager extension folder.